gk/main.c

```diff
@@ -745,6 +822,22 @@ gk_stage2(void *arg)
 	if (ret < 0)
 		goto cleanup;
 
+	if (ipv6_if_configured(&gk_conf->net->front)) {
+		int i;
+		for (i = 0; i < gk_conf->num_lcores; i++) {
+			unsigned int lcore = gk_conf->lcores[i];
+			ret = register_ipv6_acl(&gk_conf->net->front,
```

Comment:
Yes, that's correct. The problem you're pointed out is the same that we have with memory pools for mbufs. We have memory pools allocated for all sockets. You can an XXX comment saying that whenever the problem of memory pools per socket is solved, that solution solves the same issue for ACLs.